api/config.py

```diff
@@ -212,6 +223,8 @@ def initialize_db():
         upsert=True
     )
 
+    log_db.access_log.create_index('origin.id')
```

Comment:
After discussion this was out of scope. I'm going to remove the id index for now.